src/autocomplete/databases/trino/grammar/TrinoParser.g4

```diff
@@ -7,110 +7,110 @@ options {
     tokenVocab = TrinoLexer;
 }
 
+// TODO-TRINO: support multi-queries
 parse
-    : statements? EOF
-    ;
-
-statements
-    : statement SEMICOLON_?
-    | statement SEMICOLON_ statements
```

Comment:
Current Trino client doesn't work with semicolons and multiple statements, so I've removed it from our autocomplete for now for better UX

Reply:
Why do we have special logic for Trino for this? ClickHouse also doesn't support multiqueries but we have it in autocomplete
I don't think our autocomplete library should be concerned with how our websql code works, it should be independent
Also when we soon support this we will have to add it again

We should just throw error on multiple query execution like we do in ClickHouse

Reply:
I agree with Robert, need to discuss this decision

Reply:
We've discussed it. Decided to go for better UX for now